go.mod

```diff
@@ -303,7 +300,7 @@ replace github.com/hashicorp/go-version => github.com/6543/go-version v1.3.1
 
 replace github.com/shurcooL/vfsgen => github.com/lunny/vfsgen v0.0.0-20220105142115-2c99e1ffdfa0
 
-replace github.com/nektos/act => gitea.com/gitea/act v0.243.4
+replace github.com/nektos/act => gitea.com/gitea/act v0.2.51
```

Comment:
Erm… Isn't that a downgrade?

Reply:
2.51 is newer than 243.4 See: https://github.com/nektos/act/releases